docs/faq/general/why-cant-i-see-my-organization.md

```diff
@@ -4,16 +4,22 @@ description: Try these steps if you don't see your organization when adding your
 
 # Why can't I see my organization?
 
-If you don't see your organization when [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization), try the following steps:
+If you don't see your organization while [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization), please try re-adding your Git provider or refreshing the list of organizations by clicking **Add provider** or **Refresh this list** on the Organizations page:
```

Comment:
This wording feels repetitive: "organization while adding your organization", but not sure how to make it better...

Should this troubleshooting be moved to that page on "what-are-synced-organizations.md" instead?

Or maybe wording like:

"While [adding your organization on Codacy](https://deploy-preview-1051--docs-codacy.netlify.app/organizations/what-are-synced-organizations/#adding-an-organization) and it fails to appear in the page, please..."

But not sure about my suggestion either...

Reply:
Good point, here's a rewrite to make the text flow better:

```suggestion
If you can't [add your organization to Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization) because it doesn't appear on the Organizations page, please try re-adding your Git provider or refreshing the list of organizations by clicking **Add provider** or **Refresh this list** on the Organizations page:
```

What do you think @vascorsd?

Reply:
I believe it flows much better with the added SEO you say :100: